### PR TITLE
fixes #4502: Incorrect formating of LFX Terms

### LINKED ISSUE #4502: [UI] Incorrect formatting of the heading
### Description

 Incorrect formatting of the heading "Full-Time Terms" as a bullet point instead of a main heading in the [LFX programs page](https://layer5.io/programs/lfx)

### Expected Behavior

- The `Full-Time Terms` should be the heading (Not the bullet point) 
- `Spring Term: March 1st - May 31st` should be the first bullet point.
### Screenshots

image

- Reference image
image

### Environment:
- Host OS: Windows
- Browser: Chrome

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by saaagarsingh**
**Description**
Formatting of LFX terms on the website is not correct and as per requirements.
This PR fixes #
fixes: #4502 
**Notes for Reviewers**
Screenshot of Result:

issuefix4502Layer5

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d64bee5577a34e090fb204d9eb7ecea586a615c7 at: https://64a84c8213f3770c1a5906b7--layer5.netlify.app

**Comment by hirentimbadiya** [comment]
Hey @saaagarsingh , fair warning! Please wait for the issue to be assigned before creating a pull request. This ensures smoother coordination and avoids duplication of efforts. Thanks!

**Comment by saaagarsingh** [comment]
@hirentimbadiya I'm so sorry for that. I'll keep that in mind for next time and it'll never happen again.

**Comment by vishalvivekm** [comment]
@saaagarsingh Let's discuss this on websites call. Add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk), if you would :)

File: src/collections/programs/lfx-2021/lfx-2021.mdx [hunk 1]
```diff
@@ -19,7 +19,7 @@ import lfx_logo from "./Linux_Foundation_logo.svg";
 
 <div>
 <ul>
-  <li><strong>Full-Time Terms:</strong></li>
+  <li class="lfx-timeline_terms"><strong>Full-Time Terms:</strong></li>
```
Comment (Nikhil-Ladha) [inline src/collections/programs/lfx-2021/lfx-2021.mdx:22]: Move this out to be a `h4` and update the `li` below i.e, `Spring Term....` to be the header of the list.

Comment (Nikhil-Ladha) [reply src/collections/programs/lfx-2021/lfx-2021.mdx:22]: Update the same in the other files as well.

**Comment by thisiskaransgit** [comment]
@saaagarsingh, incorporate the requests, please.

**Comment by saaagarsingh** [comment]
@thisiskaransgit sure Karan,my laptop had some issues. it was given for servicing I'll get it by tomorrow and do the necessary changes to the soonest. I had no laptop that was a bit inactive these days.

**Comment by Chadha93** [comment]
> @thisiskaransgit sure Karan,my laptop had some issues. it was given for servicing I'll get it by tomorrow and do the necessary changes to the soonest. I had no laptop that was a bit inactive these days.

@saaagarsingh No updates? I'll let @thisiskaransgit or @vishalvivekm take over this PR, make suggestions on the code, and let's merge the.

**Comment by saaagarsingh** [comment]
Hey @Chadha93 it's done. I'll raise the PR today at the soonest. Sorry for the inconveniences caused due to me.

**Pull request closed without merging**